piano-hero/mobile/src/main/java/com/novoda/pianohero/SimpleNotesOutputView.java

```diff
@@ -29,54 +26,32 @@ protected void onFinishInflate() {
         messageTextView = (TextView) findViewById(R.id.simple_notes_output_text_message);
     }
 
-    public void display(Sequence sequence) {
-        checkSequenceIsSimpleElseThrow(sequence);
-
-        updateNextNoteToPlay(sequence);
-        updateMessage(sequence);
+    @Override
+    public void showRound(RoundViewModel viewModel) {
+        updateNextNoteToPlay(viewModel);
+        messageTextView.setText(viewModel.getStatusMessage());
     }
 
-    private void checkSequenceIsSimpleElseThrow(Sequence sequence) {
-        for (int i = 0; i < sequence.length(); i++) {
-            Notes notes = sequence.get(i);
-            if (notes.count() > 1) {
-                throw new IllegalArgumentException("Sequence contains chords, that's not simple enough");
-            }
-            for (Note note : notes.notes()) {
-                if (simplePitchNotationFormatter.format(note).endsWith("#")) {
-                    throw new IllegalArgumentException("Sequence contains sharps, that's not simple enough");
-                }
-            }
+    private void updateNextNoteToPlay(RoundViewModel viewModel) {
+        for (String note : viewModel) {
+            nextNoteToPlayTextView.setText(note);
         }
     }
 
-    private void updateNextNoteToPlay(Sequence sequence) {
-        int nextNotesPosition = sequence.position();
-        for (Note note : sequence.get(nextNotesPosition)) {
-            updateNextNoteToPlayTextViewWith(note);
-        }
+    @Override
+    public void showGameComplete() {
+        PianoHeroApplication.popToast("game complete, another!");
```

Comment:
this message should be passed as a parameter - the presenter is the only one that knows that you're starting a new game onGameComplete